new --strip option to build bundle (#276)
luvit/luvi: README.md

luvi myapp
# Build the binary when done
luvi myapp -o mybinary

> # Build the binary with compiled, striped Lua bytecode when done
> luvi myapp -o mybinary -s
> # Build the binary with compiled Lua bytecode when done
> luvi myapp -o mybinary --compile

Reviewer: ```suggestion
# Build the binary with compiled Lua bytecode
luvi myapp -o mybinary --compile
# Build the binary with compiled and stripped Lua bytecode
luvi myapp -o mybinary --strip
```

  bundle            Path to directory or zip file containing bundle source.
                    `bundle` can be specified multiple times to layer bundles
                    on top of each other.
  --version         Show luvi version and compiled in options.
  --output target   Build a luvi app by zipping the bundle and inserting luvi.
  --main path       Specify a custom main bundle path (normally main.lua)
  --compile         Compile Lua code into bytecode before bundling.

>   --strip           Strip debug info from compiled Lua code.

Reviewer: ```suggestion
  --strip           Compile Lua code and strip debug info.
```
